test/check-application

```diff
@@ -2028,15 +2031,28 @@ class TestApplication(testlib.MachineCase):
     def testHealthcheckUser(self):
         self._testHealthcheck(False)
 
-    def testPodmanRestartEnabled(self):
+    @testlib.skipImage("podman-restart not available in debian/ubuntu", *DISTROS_WITHOUT_PODMAN_USER_RESTART)
```

Comment:
It does work fine in Debian, right? so somehow the package gets misbuilt in Ubuntu.